src/webviews/apps/home/home.html

```diff
@@ -396,6 +396,14 @@ <h2 class="nav-list__title t-eyebrow">Configuration</h2>
 					><code-icon class="nav-list__icon" icon="link"></code-icon
 					><span class="nav-list__label">Autolinks Settings</span></a
 				>
+				<a
```

Comment:
Let's do the following:
- Move the GitLens settings to the top, above Popular Views -- @d13 what do you think?
- Create a new Integrations section and put integrations first and autolinks 2nd, and place if after the activity bar section

Reply:
I like it